pkgs/games/gargoyle/default.nix

```diff
@@ -20,13 +20,13 @@ let
 in
 
 stdenv.mkDerivation {
-  name = "gargoyle-2018-10-06";
+  name = "gargoyle-2019-01-01";
 
   src = fetchFromGitHub {
     owner = "garglk";
     repo = "garglk";
-    rev = "d03391563fa75942fbf8f8deeeacf3a8be9fc3b0";
-    sha256 = "0icwgc25gp7krq6zf66hljydc6vps6bb4knywnrfgnfcmcalqqx9";
+    rev = "28b999f83542deee60f2447e4b6109b6db358f57";
```

Comment:
```suggestion
    rev = version;
```

Reply:
I think I must have implemented some suggestion wrong, but I get:

    error: fetchurl requires a hash for fixed-output derivation: https://github.com/garglk/garglk/archive/2019.1.1.tar.gz

Reply:
You removed the `sha256` attribute.

Reply:
@emacsomancer let me know if you need any help. I can also update the PR to make these changes if you prefer.

Reply:
thanks. I've pushed the changes now.